docs/onboarding/8_16/operator/onprem_kube_stack_values.yaml

```diff
@@ -33,6 +33,12 @@ defaultCRConfig:
           name: elastic-secret-otel
           key: elastic_api_key
 
+clusterRole:
```

Comment:
there is this warning - it is not relevant for the not EKS envs, but on EKS collectors should have access to the configmaps:
```
2024-09-24T12:34:19.651Z	warn	internal/resourcedetection.go:130	failed to detect resource	{"kind": "processor", "name": "resourcedetection/eks", "pipeline": "logs", "error": "isEks() error retrieving auth configmap: failed to retrieve ConfigMap kube-system/aws-auth: configmaps \"aws-auth\" is forbidden: User \"system:serviceaccount:kube-system:opentelemetry-kube-stack-cluster-stats-collector\" cannot get resource \"configmaps\" in API group \"\" in the namespace \"kube-system\""}
```

Reply:
We can also add it this block in comments with a note that uncomented it in case of EKS . Just an idea

Reply:
I would keep it not commented - so the experience for cloud providers do not require values customization